### PR TITLE
[skin.estuary/pvr] Add deleted to PVR Recordings breadcrumbs and increase width of info panel to fit 12 hour clock

### PR DESCRIPTION
**Pull request by phunkyfish**
## Description

When viewing deleted recordings on PVR Recordings UI there is nothing on screen to signify you are looking at deleted vs undeleted recordings.

Also, if you use 12 hour clock timer description will not fit on one line if both times use all four time digits (HH:MM), i.e. 11:50 AM and 12:45 PM.

## Motivation and Context

You can essentially get stuck in the deleted recordings view without knowing where you are.

## How Has This Been Tested?

Runtime tested on OSX.

## Screenshots (if appropriate):

Deleted breadcrumb: 

Screenshot 2019-10-08 at 16 46 10

Time date fit on one line:

Screenshot 2019-10-08 at 16 46 53

## Types of change

- [] **Bug fix** (non-breaking change which fixes an issue)
- [ ] **Clean up** (non-breaking change which removes non-working, unmaintained functionality)
- [X] **Improvement** (non-breaking change which improves existing functionality)
- [ ] **New feature** (non-breaking change which adds functionality)
- [ ] **Breaking change** (fix or feature that will cause existing functionality to change)
- [ ] **Cosmetic change** (non-breaking change that doesn't touch code)
- [ ] **None of the above** (please explain below)

## Checklist:

- [X] My code follows the **[Code Guidelines](https://github.com/xbmc/xbmc/blob/master/docs/CODE_GUIDELINES.md)** of this project 
- [ ] My change requires a change to the documentation, either Doxygen or wiki
- [ ] I have updated the documentation accordingly
- [X] I have read the **[Contributing](https://github.com/xbmc/xbmc/blob/master/docs/CONTRIBUTING.md)** document
- [ ] I have added tests to cover my change
- [ ] All new and existing tests passed

### CODE REVIEW

File: addons/skin.estuary/xml/Variables.xml [hunk 1]
```diff
@@ -383,7 +383,9 @@
 		<value>$LOCALIZE[19021] / $INFO[Control.GetLabel(29)] - $INFO[Control.GetLabel(30)]</value>
 	</variable>
 	<variable name="BreadcrumbsPVRRecordingsVar">
-		<value condition="Window.IsActive(TVRecordings)">$LOCALIZE[19020] / $LOCALIZE[19017]$INFO[Control.GetLabel(30), / ]</value>
+		<value condition="Window.IsActive(TVRecordings) + String.Contains(Control.GetLabel(7),*)">$LOCALIZE[19020] / $LOCALIZE[19017]$INFO[Control.GetLabel(30), / ] - $LOCALIZE[19179]</value>
+		<value condition="Window.IsActive(TVRecordings) + !String.Contains(Control.GetLabel(7),*)">$LOCALIZE[19020] / $LOCALIZE[19017]$INFO[Control.GetLabel(30), / ]</value>
+		<value condition="Window.IsActive(RadioRecordings) + String.Contains(Control.GetLabel(7),*)">$LOCALIZE[19021] / $LOCALIZE[19017]$INFO[Control.GetLabel(30), / ] - $LOCALIZE[19179]</value>
```
Comment (ksooo) [inline addons/skin.estuary/xml/Variables.xml]: Out of curiosity: Can you explain this `GetLabel(7)`voodoo to me? Is this a special label and if yes, what is its purpose and where does its value come from?

Comment (HitcherUK) [reply addons/skin.estuary/xml/Variables.xml]: Used to determine if a radio button is on or off.

When on they will return `*`

Comment (phunkyfish) [reply addons/skin.estuary/xml/Variables.xml]: 7 is the ID of the radio button for deleted recordings. It comes from Includes_MediaMenu.xml.

Comment (ksooo) [reply addons/skin.estuary/xml/Variables.xml]: Ah, thanks. Seems a bit hacky to me, but my skinning skills a very basic.

**Review by ksooo — APPROVED** [review]

**Pull request merged**